src/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -308,7 +307,15 @@ export const BuyerGuaranteeIndex: FC = () => {
             millions of global businesses.
           </Text>
 
-          <PoweredByStripeIcon width={150} height={34} mt={2} />
+          <Spacer y={2} />
+
+          <img
+            src="https://files.artsy.net/images/PoweredByStripe_1.svg"
```

Comment:
SVGs that don't belong in the system just get uploaded on tools.artsy.net and included via `img` tag.